content/docs/user-guide/dvc-files.md

```diff
@@ -161,22 +161,27 @@ stages:
     # User metadata and comments are supported.
 ```
 
-`dvc.yaml` files consists of a group of `stages` with names provided explicitly
-by the user with the `--name` (`-n`) option of `dvc run`. Each stage can contain
+💡 You can [parameterize this file](#parameterize-dvcyaml) with the special
```

Comment:
we should have this references, there should be some general description, then basic layout, then other syntax like templating, loops, etc. Again, let's refer to some good examples that describe languages. (Btw, templating is a way better term in our case than parametrization to my mind)

Reply:
Well for now this is all still in guides so it's still not "right" that we have references here. We could have the dvc.yaml "langauge" ref. as its own section (before Command Reference, for example but that change I would definitely try to avoid putting here (let's just prioritize #2059 to right after this?)

I still need to do that research on other tech refs and guides ⌛

Reply:
BTW I removed those 💡 notes from the main dvc.yaml page (just added a gray note linking to the Advanced dvc.yaml page). We can keep that `${model_file}` in the example right? No strong opinion but it could spark people's curiosity (to find the advanced page).